proxy/backend/Dockerfile

```diff
@@ -15,7 +15,7 @@ RUN ./gradlew --no-daemon shadowJar
 FROM eclipse-temurin:11
 WORKDIR /usr/src/app
 COPY --from=build /usr/src/app/build/libs/openlineage-proxy-*.jar /usr/src/app
-COPY proxy.dev.yml proxy.dev.yml
+COPY proxy.yml proxy.yml
 COPY docker/entrypoint.sh entrypoint.sh
-EXPOSE 5000 5001
+EXPOSE 6000 6001
```

Comment:
any particular reason for this change?

Reply:
Yes, Marquez is configured to work on port 5000, so I changed this port so one can use Marquez to give it a try.